plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraTypeManager.java

```diff
@@ -11,6 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Revert unrelated change.

Reply:
Sorry, didn't quite understand this one.. What unrelated change?

Reply:
This empty line didn't exist before this commit.

Reply:
@ebyhr Got it, removed :)